arts/observation.py

```diff
@@ -0,0 +1,58 @@
+from arts.telescope import Telescope
+
+
+class Observation(Telescope):
```

Comment:
Maybe the Observation should take an instance or list of instances of the Telescope class?

This way, we can run a single observation for multiple telescopes?